frontend/js/main.js

```diff
@@ -1605,35 +1670,34 @@ const readTextFile = (file, callback) => {
+  let sessionContent = '';
   if (sessionArray === null) {
-    document.getElementById('session-table-content').innerHTML = `<tr>
-      <td colspan='5' class='center session-image'><img src='Icons/Icons/Group 2154.svg' alt='nothing to show'></td>
-    </tr>
-    <tr>
-      <td colspan='5' class='center simple-grey-text'>No active session available! <br> Please connect a database to initiate a new session.</td>
-    </tr>`
+    if (document.getElementById('session-table-content')) {
+      sessionContent = `<tr>
+                          <td colspan='5' class='center session-image'><img src='Icons/Icons/Group 2154.svg' alt='nothing to show'></td>
+                        </tr>
+                        <tr>
+                          <td colspan='5' class='center simple-grey-text'>No active session available! <br> Please connect a database to initiate a new session.</td>
+                        </tr>`;
+    }
   }
   else {
     let sessionArrayLength = sessionArray.length;
-    let sessionTableContentEle = document.getElementById('session-table-content');
-    let sessionTableContent = '';
     for (var x = 0; x < sessionArrayLength; x++) {
       let session = sessionArray[x];
-      // let driver = session.driver;
-      // let path = session.path;
       let sessionName = session.fileName;
       let sessionDate = session.createdAt.substr(0, session.createdAt.indexOf("T"));
       let sessionTime = session.createdAt.substr(session.createdAt.indexOf("T") + 1);
-      sessionTableContent += `<tr class='d-flex'>
-                                <td class='col-2 session-table-td2'>${sessionName}</td>
-                                <td class='col-4 session-table-td2'>${sessionDate}</td>
-                                <td class='col-2 session-table-td2'>${sessionTime}</td>
-                                <td class='col-4 session-table-td2 session-action' id=${x}>
-                                  <a href='#/schema-report-resume-session' style='cursor: pointer; text-decoration: none;' onclick='resumeSessionHandler(${x}, ${JSON.stringify(sessionArray)})'>Resume Session</a>
-                                </td>
-                              </tr>`;
-      sessionTableContentEle.innerHTML = sessionTableContent;
+      sessionContent += `<tr class='d-flex'>
+                          <td class='col-2 session-table-td2'>${sessionName}</td>
+                          <td class='col-4 session-table-td2'>${sessionDate}</td>
+                          <td class='col-2 session-table-td2'>${sessionTime}</td>
+                          <td class='col-4 session-table-td2 session-action' id=${x}>
+                            <a href='#/schema-report-resume-session' style='cursor: pointer; text-decoration: none;' onclick='resumeSessionHandler(${x}, ${JSON.stringify(sessionArray)})'>Resume Session</a>
+                          </td>
+                        </tr>`;
     }
+    return sessionContent;
```

Comment:
```
return sessionContent;
```
This return statement should be outside of if else section, other wise it will not work for the if case

Reply:
this code change is done